.github/workflows/ci-repo2docker.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install jupyter-repo2docker
+          python -m pip install six
```

Comment:
It looks like this is due to an upstream issue in Docker's Python bindings (xref https://github.com/docker/docker-py/issues/2842). Could you add a comment which links back to that `docker/docker-py` issue so we know when we can remove this `pip install six` step?

Reply:
I've forgotten to do this, apologies! I've added the comment and also added chardet until it get fixed in repo2docker 😄